### PR TITLE
LPS-178405 Add CanCreateActionAddAnObjectEntryUsingOldValueWithOnAfterDeleteTrigger test

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LPS-178405

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -192,6 +192,81 @@ definition {
 			actionLabel = "Action Label");
 	}
 
+	@description = "LPS-178405 - Verify if actions works with oldValue function when trigger is On After Delete (Add an Object Entry)"
+	@priority = 5
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:196]: Bump this down to priority 4 per ticket description

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 2]
```diff
@@ -192,6 +192,81 @@ definition {
...
+	test CanCreateActionAddAnObjectEntryUsingOldValueWithOnAfterDeleteTrigger {
+		task ("Given: Two custom objects with a text field") {
+			for (var objectNum : list "1,2") {
+				ObjectAdmin.addObjectViaAPI(
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:200]: Can you add the ticket number as the custom object name. So like "Custom Object 178405-${objectNum)"

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 3]
```diff
@@ -192,6 +192,81 @@ definition {
...
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 1");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				active = "true",
+				fieldName = "Custom Text Field",
+				inputExpression = "customObjectField == 'Entry Test'",
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:228]: the inputExpression needs to have the oldValue function here. 

Should look something like _oldValue("customObjectField") == "Entry Test"_

**Review by epark18 — COMMENTED** [review]
@nkkwon This one is solid, it just needs the change to include oldValue. The rest of the test should be unaffected

**Comment by timpak** [comment]
RIP looks like we need to resolve another merge conflict, @nkkwon

**Comment by timpak** [comment]
@nkkwon when you have a chance, can you fix the merge conflict and forward this? If you don't have bandwidth, let me know.

**Comment by nkkwon** [comment]
Hey @timpak sorry currently don't have the bandwidth right now

**Comment by timpak** [comment]
No worries, @nkkwon. 

@Luizcarlosqueiroz can you get one of the SDG guys to fetch this, rebase the branch to fix the merge conflicts, and forward it?

**Pull request closed without merging**